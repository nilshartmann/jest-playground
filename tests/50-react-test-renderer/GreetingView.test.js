import React from 'react';
import renderer from 'react-test-renderer';

import Editor from './Editor';

test('it should bla', () => {
        const tree = renderer.create(
        <Editor/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
