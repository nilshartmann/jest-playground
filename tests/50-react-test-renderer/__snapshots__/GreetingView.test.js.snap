// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should bla 1`] = `
<div>
  Eingabe:
  <input
    onChange={[Function]}
    type="text"
    value={undefined}
  />
</div>
`;
